lib/src/bloc/action_bloc.dart

```diff
@@ -1,6 +1,21 @@
 import 'package:hooked_bloc/src/mixins/bloc_action_mixin.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
+/// Syntax sugar for [Bloc] with [BlocActionMixin]
+///
+/// [BlocActionMixin] can be useful when there is existing Cubit
```

Comment:
Maybe consider more straightforward sentence?
`[BlocActionMixin] is useful while adding events to the existing Cubits`